echopype/convert/set_groups_azfp.py

```diff
@@ -37,6 +37,51 @@ class SetGroupsAZFP(SetGroupsBase):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
+        # get frequency values
+        freq_old = list(self.parser_obj.unpacked_data["frequency"])
+
+        # sort the frequencies in ascending order
+        freq_new = freq_old[:]
+        freq_new.sort(reverse=False)
+
+        # obtain sorted frequency indices
+        self.freq_ind_sorted = [freq_new.index(ch) for ch in freq_old]
+
+        # obtain sorted frequencies
+        self.freq = self.parser_obj.unpacked_data["frequency"][self.freq_ind_sorted]
```

Comment:
recommend changing this to `self.freq_sorted` so that it is explicit that it is sorted like how you have it in `freq_ind_sorted`.

Reply:
Good point, I will change this name.